docs/fsharp/language-reference/compiler-messages/fs0052.md

```diff
@@ -0,0 +1,22 @@
+﻿---
+title: "Compiler error FS0052"
+ms.date: 12/11/2020
+f1_keywords:
+  - "FS0052"
+helpviewer_keywords:
+  - "FS0052"
+---
+
+# FS0052: Defensive copy
+
+In an F# project, the default `<WarningLevel>` is 3. This warning appears when the warning level is set to its maximum value, 5. In such a case, this code:
+
+[!code-fsharp[FS0052-warn](~/samples/snippets/fsharp/compiler-messages/fs0052.fsx#L2)]
+
+Will yield this warning:
+`The value has been copied to ensure the original is not mutated by this operation or because the copy is implicit when returning a struct from a member and another member is then accessed`
```

Comment:
```suggestion
> The value has been copied to ensure the original is not mutated by this operation or because the copy is implicit when returning a struct from a member and another member is then accessed
```